src/lib/y2security/selinux.rb

```diff
@@ -122,21 +188,30 @@ def save
       return false unless configurable?
 
       Yast::Bootloader.modify_kernel_params(mode.options)
+      config_file.selinux = mode.id.to_s
+      config_file.save
 
       return true if Yast::Mode.installation
 
       Yast::Bootloader.Write
     end
 
+    # Returns needed patterns defined in the product features
+    #
+    # @return [String] defined patterns in product features to have SELinux working as expected
+    def needed_patterns
```

Comment:
What does it return if there is no definition ?

Reply:
It returns and empty string, if I'm not wrong

Reply:
Now it returns an empty array. See https://github.com/yast/yast-security/pull/87#discussion_r574408651 and https://github.com/yast/yast-security/pull/87/commits/9191e1564ae39466ebe982dc2760fa20c1bec0a8